ecc/enclave/enclave_stub.go

```diff
@@ -389,6 +391,22 @@ func (e *StubImpl) Destroy() error {
 	return nil
 }
 
-func (e *StubImpl) MrEnclave() string {
-	return MrEnclave
+// Loads the MrEnclave string from file
+func LoadMrEnclave() error {
+	binMrEnclave, err := ioutil.ReadFile("mrenclave")
+	if err != nil {
+		return fmt.Errorf("Error reading MrEnclave from file")
+	}
+	MrEnclave = string(binMrEnclave)
+	return nil
+}
+
+func (e *StubImpl) MrEnclave() (string, error) {
```

Comment:
since MrEnclave() is only invoked once during init ... why not reading here from file?